src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1394,6 +1402,8 @@
     <Resource Include="UI\Images\profile_normal.png" />
   </ItemGroup>
   <ItemGroup>
+    <EmbeddedResource Include="Views\GuidedTour\HtmlPages\learnPackages.en-US.html" />
```

Comment:
I dont think we duplidate `en-US` version for `DocumentationBrowserViewExtension`, unless I missed anything?

Reply:
Yes, it's not necessary. I left this as an example, removing now. Thanks!